src/pages/create-course/CreateCourse.tsx

```diff
@@ -10,12 +10,17 @@ import { authRoutes } from '~/router/constants/authRoutes'
 import { ButtonTypeEnum, ButtonVariantEnum } from '~/types'
 
 import { styles } from '~/pages/create-course/CreateCourse.styles'
+import AddCourseBanner from '~/containers/add-course-banner/AddCourseBanner'
+import React from 'react'
```

Comment:
organize imports, and importing React is redundant

Reply:
Done